src/js/containers/covid19/Covid19Container.jsx

```diff
@@ -174,25 +175,33 @@ const Covid19Container = () => {
             </StickyHeader>
             <LoadingWrapper isLoading={isLoading}>
                 <main id="main-content" className="main-content usda__flex-row">
-                    <div className="sidebar usda__flex-col">
-                        <Sidebar
-                            pageName="covid19"
-                            fixedStickyBreakpoint={scrollPositionOfSiteHeader(Cookies.get('usaspending_covid_homepage'))}
-                            active={activeSection}
-                            jumpToSection={jumpToCovid19Section}
-                            detectActiveSection={setActiveSection}
-                            sections={Object.keys(componentByCovid19Section())
-                                .filter((section) => componentByCovid19Section()[section].showInMenu)
-                                .map((section) => ({
-                                    section: snakeCase(section),
-                                    label: componentByCovid19Section()[section].title
-                                }))} />
+                    <div className="sidebar__wrapper">
```

Comment:
I would swap these class names to `sidebar` wrapping something like `sidebar__content` to adhere to BEM. (`sidebar` as the block and `sidebar__content` as an element of that block)